index.html

```diff
@@ -1,12 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Cruise Ships</title>
-    <link rel="stylesheet" href="css/normalize.css" />
-    <link rel="stylesheet" href="css/style.css" />
-  </head>
-  <body></body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Cruise Ships</title>
+        <link rel="stylesheet" href="css/normalize.css" />
+        <link rel="stylesheet" href="css/style.css" />
+    </head>
+    <body>
+        <div id="hud"></div>
+        <div id="viewport">
+            <button id="sailbutton">Set Sail!</button>         
+            <div id="ports">
+            </div>
+            <div id="ship"></div>
+        <script src="src/controller.js"></script>
+        <script src="src/port.js"></script>
+        <script src="src/itinerary.js"></script>
+        <script src="src/ship.js"></script>
+        <script>
+            const itinerary = new Itinerary([
```

Comment:
It would be nice if you could add a form to the page so these values could be changed, rather than just hardcoded as it is right now